package.json

```
@@ -15,7 +15,8 @@
     "release": "changelogen --release && git push --follow-tags",
     "lint-staged": "lint-staged",
     "format": "prettier --write .",
-    "prepare": "husky"
+    "prepare": "husky",
+    "format:check": "prettier --check ."
```

```suggestion
    "format:check": "prettier --check .",
    "prepare": "husky"
```
Always add new line before last one to avoid extra diffs